src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,58 @@
+package command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeException;
+import task.Task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
```

Comment:
Perhaps you may consider writing a Javadoc header comment describing the class here, since the coding standard mentions that "You MUST write header comments for all classes, public methods except for getters/setters."

I notice that there is no Javadoc comment for your other classes as well, so you may want to consider writing header comments for all the classes.